src/robot/variables/typeconverters.py

```diff
@@ -122,13 +124,33 @@ def _literal_eval(self, value, expected):
         return value
 
 
+@TypeConverter.register
+class StringConverter(TypeConverter):
+    type = str
+    type_name = 'string'
+    aliases = ('str',)
+
+    def _convert(self, value, explicit_type=True):
+        return str(value)
+
+
```

Comment:
What is this class for? It seems to register a converter for strings, but shouldn't something like that be a generic enhancement deserving its own PR and also an issue?

Notice also that due to differences between Python 2 and 3, this code doesn't work correctly. `str` is actually bytes in Python 2 and there `unicode` should be used instead. Importantly, `str(value)` fails with if the `value` has a non-ASCII string representation.